src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+
+**[a8] Legend Swatches**
+
+For a given coloring (see [a1] above) the set of values observed for this coloring across the tree is displayed as a swatch. This is responsive — only the currently in view part of the tree is considered, so as you filter the data and zoom the tree the swatches will update. For continuous and temporal scales, the values are grouped into bins automatically (unless specified in the scale).
+
+**[a9] What’s shown when clicking on a node?**
+
+For the selected node Auspice displays the following information, as applicable:
+
+1. Strain name,  as defined by `node.name`.
+2. Vaccine information (`node.node_attrs.vaccine`), if applicable.
+3. Sample date (`node.node_attrs.num_date`) as well as any associated confidence values. For internal nodes this is always labelled as “Inferred date”;  terminal nodes are labelled as “Inferred collection date” if a non-zero uncertainty range is provided, otherwise “Collection date”. 
+4. Publication info, if provided. This considers  `node_attrs.author: Dictionary` and its associated keys `value`, `title`, `paper_url`, `journal` (only `value` is required). 
+5. Strain information. This considers `node_attrs.gisaid_epi_isl`, `node_attrs.genbank_accession`, `node_attrs.accession` and `node_attrs.url` [2].
+6. Any other `node_attrs` provided are displayed. If the attribute is also a colouring then the colouring title will be used. Note that associated confidence values are not displayed.
+7. `branch_attrs.mutations` are displayed.
+    * For terminal nodes, we traverse the tree to display the total mutations between the root node and this tip. This allows us to categorise mutations as “Changes” or “Reversions to root”.  Note that (e.g.) a series of mutations such as “A -> T -> C” at position 100 will be shown as “A100C”. 
+    * For internal nodes we show the mutations ascribed to the branch but categorise them as “Unique”, “Homoplasies” and “Reversions to root”. Note that the first two categories are mutually exclusive however a mutation may appear again in the “Reversions to root” category, as applicable.
+
+
+**[a10] listed filters in the footer of the page**
+
+Filters in the bottom of the page are specified by   `meta.filters: Array<string>`. The values should correspond to keys in `node_attrs`, and auspice traverses the tree to find the set of all values. If a colouring was provided for this key then Auspice will attempt to display the title rather than the key. Selecting filters here behaves the same was as sidebar data filtering (see [a2], above).
+
+---
+
+### Exporting data via Augur
+
+We now consider how information flows through Augur, specifically `augur export v2`  which produces the dataset (Auspice) JSON described above. Augur has two main types of “metadata”  which we’ll refer to as “TSV-data” (provided with the `--metadata` argument) and “node-data” which are JSONs provided via `—node-data`. 
```

Comment:
Suggest referencing the "[Data formats](https://docs.nextstrain.org/en/latest/reference/data-formats.html)" doc for the node data files.

Also, minor typographical error:

```suggestion
We now consider how information flows through Augur, specifically `augur export v2`  which produces the dataset (Auspice) JSON described above. Augur has two main types of “metadata”  which we’ll refer to as “TSV-data” (provided with the `--metadata` argument) and “node-data” which are JSONs provided via `--node-data`. 
```

Reply:
Done and done